### PR TITLE
Translations update from Mattermost Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Mattermost Weblate](https://translate.mattermost.com) for [Mattermost/mattermost-mobile-v2](https://translate.mattermost.com/projects/mattermost/mattermost-mobile-v2/).

Current translation status:

Weblate translation status

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by ctlaltdieliet — APPROVED** [review]
lgtm :rocket:

**Comment by ctlaltdieliet** [comment]
@lieut-data 
Can you see why the tests are failing?
Thank you

**Comment by lieut-data** [comment]
Looks to be an issue with previous commits -- @yasserfaraazkhan, do you know what might be causing this?

File: assets/base/i18n/sv.json [hunk 1]
```diff
@@ -96,13 +96,39 @@
   "camera_type.video.option": "Spela in video",
   "center_panel.archived.closeChannel": "Stäng kanalen",
   "channel_add_members.add_members.button": "Lägg till medlemmar",
+  "channel_bookmark.add.detail_title": "Titel",
+  "channel_bookmark.add.emoji": "Lägg till emoij",
```
Comment (phieri) [inline assets/base/i18n/sv.json:100]: Letters flipped in emoji.

**Review by phieri — CHANGES_REQUESTED** [review]

File: assets/base/i18n/sv.json (hunk 1 above)
Comment (ctlaltdieliet) [reply assets/base/i18n/sv.json]: Sharp eye @phieri !
Fixed it through Weblate and pushed the changes

**Pull request merged**